examples/vector-esri-edit.js

```diff
@@ -97,8 +95,12 @@ selected.on('remove', function(evt) {
       featureProjection: map.getView().getProjection()
     }) + ']';
     var url = serviceUrl + layer + '/updateFeatures';
-    $.post(url, { f: 'json', features: payload }).done(function(data) {
-      var result = JSON.parse(data);
+    var data = new FormData();
```

Comment:
I think this wll not work in all target browser, right? We still support IE9, for example. https://developer.mozilla.org/docs/Web/API/FormData